lib/omniauth/login_dot_gov/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module LoginDotGov
-    VERSION = '0.1.2'.freeze
+    VERSION = '0.1.3'.freeze
```

Comment:
just checking, this is a change to the internal API, not any external API changes when customers use this gem right?

Since this is an important change to check for, WDYT of bumping the major version?

Reply:
This will be a breaking change if customers update the gem but don't update their logout code. So yes, I can see the argument for making this be v0.2.0.